dronecan_gui_tool/main.py

```diff
@@ -41,6 +42,13 @@
 logging.basicConfig(stream=sys.stderr, level=logging_level,
                     format='%(asctime)s %(levelname)s %(name)s %(message)s')
 
+from .version import __version__
```

Comment:
Perhaps this could go above the debug args, given this doesnt make use of logging at all.